Aptoide/appcoins-wallet-android
# [WAL-67] Uniformize edit texts

**What does this PR do?**

Uniformizes edit texts to use the same style

**Database changed?**

No

**Where should the reviewer start?**

styles.xml

**How should this be manually tested?**

Check if there are more views affected by this that were not changed.
Check if the views that were changed have any problem

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/WAL-67

**Code Review Checklist**
